exam_1/task1/main.c

```diff
@@ -0,0 +1,99 @@
+#include <stdio.h>
+#include <string.h>
+
+int tripleTransformation(char *binary, char *octale)
+{
+    int result = 0;
+    int length = strlen(binary);
+    if (length != 3)
+    {
+        return -1;
+    }
+    for (int i = 2; i >= 0; --i)
+    {
+        if (binary[i] != '1' && binary[i] != '0')
+        {
+            return -1;
+        }
+        if (binary[2 - i] == '1')
+        {
+            int exponent = i;
+            int sum = 1;
```

Comment:
```suggestion
            int transformedBit = 1;
```
Это не сумма, это преобразованное значение бита двоичного числа
(-0.25)